pkgs/tools/misc/tmux-sessionizer/default.nix

```
@@ -10,7 +10,8 @@
 }: let
 
   name = "tmux-sessionizer";
-  version = "0.3.2";
+  # v0.4.1 is not released yet, but v0.4.0 has version discrepancy between Cargo.toml and Cargo.lock and does not build
+  version = "unstable-2024-02-06";
```

These unstable versions are usually considered to be a version of their own - you should update the commit and PR title to reflect this (`tmux-sessionizer: 0.3.2 -> 0.4.0-unstable-2024-02-06`)

Also, by https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming we should include the last stable version.

```suggestion
  version = "0.4.0-unstable-2024-02-06";
```